aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java

```diff
@@ -122,6 +122,17 @@ private DescribePatchBaselinesResponse describePatchBaselinesResponse(final Stri
                 .stream().map(baseline -> baseline.baselineId()).collect(Collectors.toList());
 
         for (String baselineId : baselineIdList) {
+
+            System.out.println("*****Baseline deatils*******");
+            System.out.println(baselineId);
+
+            if(baselineId.contains("/"))
+            {
+                  String baselineArr[]=  baselineId.split("/");
+                baselineId = (baselineArr.length >=1) ? baselineArr[1] : "";
+
+            }
+            System.out.println("*****xxxxe*******");
```

Comment:
please remove those comments

Reply:
Removed the log print statments and pushed the changes to branch